render/mapping.go

```diff
@@ -180,11 +179,20 @@ func MapProcess2Container(n RenderableNode) (RenderableNode, bool) {
 		return n, true
 	}
 
+	// Don't propogate non-internet pseudo nodes
+	if n.Pseudo {
+		return n, false
+	}
+
 	// Otherwise, if the process is not in a container, group it
-	// into an "Uncontained" node
+	// into an per-host "Uncontained" node
 	id, ok := n.NodeMetadata[docker.ContainerID]
-	if !ok || n.Pseudo {
-		return newDerivedPseudoNode(UncontainedID, UncontainedMajor, n), true
+	if !ok {
+		hostID := report.ExtractHostID(n.NodeMetadata)
```

Comment:
They'll all get grouped into a single uncontained node, if they don't have a host id in their nodemetadata.